Code/GraphMol/SubstructLibrary/PatternFactory.h

```diff
@@ -19,7 +19,13 @@ namespace RDKit {
 /*
      \param sslib The substructure library (without pattern fingerprints)
      \param numThreads the number of threads to use, -1 for all threads [default 1]
+     \param patterns if supplied, use this pattern holder and take ownership [default nullptr]
 */
-RDKIT_SUBSTRUCTLIBRARY_EXPORT void addPatterns(SubstructLibrary &sslib, int numThreads=1);  
-}
+RDKIT_SUBSTRUCTLIBRARY_EXPORT void addPatterns(SubstructLibrary &sslib,
```

Comment:
Please swap the order of these two declarations. The current ordering ends up making the documentation wrong.